Makefile

```diff
@@ -85,4 +85,5 @@ release-files:
 generated-files: release-files
 	(cd openshift-knative-operator; ./hack/update-codegen.sh; ./hack/update-deps.sh; ./hack/update-manifests.sh)
```

Comment:
Hmm, now I'm thinking maybe we should write this as:

```bash
cd openshift-knative-operator && ./hack/update-codegen.sh && ./hack/update-deps.sh && ./hack/update-manifests.sh
```
WDYT @markusthoemmes? If yes, we could do it in follow up.